graphql_compiler/tests/test_macro_expansion.py

```diff
@@ -38,6 +38,60 @@ def test_macro_edge_basic(self):
         compare_graphql(self, expected_query, expanded_query)
         self.assertEqual(expected_args, new_args)
 
+    def test_macro_edge_pro_forma_fields(self):
```

Comment:
Can you make a comment explaining what a pro_format field here is and in the test below?